content/docs/setup/kubernetes/multicluster-install/gateways/index.md

```diff
@@ -148,8 +189,19 @@ spec:
   # to this address.
   - address: <IPofCluster2IngressGateway>
     ports:
-      http1: 15443 # Do not change this port value
-      tcp2: 15443 # Do not change this port value
+      http1: 15443
+      tcp2: 15443
+{{< /text >}}
+
+__Note__: If your Kubernetes cluster is running in an environment that does not
+support external load-balancers, use the following commands to obtain the correct
+`endpoints.address` and `http1`/`tcp2` port values of the remote `istio-ingressgateway`
+service:
+
+{{< text bash >}}
+$ NODE=$(kubectl get ep/istio-ingressgateway -n istio-system -o=jsonpath='{.subsets[0].addresses[0].nodeName}')
+$ kubectl get nodes/$NODE -o=jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}'
```

Comment:
Does this get the node followed by get the IntrernalIP address work on any platfrom where there is no external load balancer? If not, we probably shouldn't document it. If it does, we should use the same approach in the Ingress task, instead of listing all the separate platform-specific instructions: http://localhost:1313/docs/tasks/traffic-management/ingress/#determining-the-ingress-ip-and-ports-when-using-a-node-port

Reply:
I tested the commands on minikube. I don't know if they will work on all platforms, so I will remove the commands and leave the note.